### PR TITLE
Final changes before branching stable/3.1.x.

### PR DESCRIPTION
**Pull request by felixxm**

### CODE REVIEW

File: docs/releases/3.1.txt [hunk 1]
```diff
@@ -102,19 +102,14 @@ Minor features
...
-
 :mod:`django.contrib.auth`
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 * The default iteration count for the PBKDF2 password hasher is increased from
   180,000 to 216,000.
 
-* Added the :setting:`PASSWORD_RESET_TIMEOUT` setting to define the number of
-  seconds a password reset link is valid for. This is encouraged instead of
+* The new :setting:`PASSWORD_RESET_TIMEOUT` setting allows to define the number
```
Comment (pope1ni) [inline docs/releases/3.1.txt:111]: `allows to define` → `allows defining`

File: docs/releases/3.1.txt [hunk 2]
```diff
@@ -102,19 +102,14 @@ Minor features
...
+  of seconds a password reset link is valid for. This is encouraged instead of
```
Comment (pope1ni) [inline docs/releases/3.1.txt:112]: `of deprecated PASSWORD_RESET_TIMEOUT_DAYS` → `of the deprecated PASSWORD_RESET_TIMEOUT_DAYS setting`

File: docs/releases/3.1.txt [hunk 3]
```diff
@@ -170,7 +160,7 @@ Minor features
   :class:`~django.contrib.postgres.fields.RangeField`.
 
 * The new :lookup:`rangefield.lower_inc`, :lookup:`rangefield.lower_inf`,
-  :lookup:`rangefield.upper_inc`, and :lookup:`rangefield.upper_inf` allows
+  :lookup:`rangefield.upper_inc`, and :lookup:`rangefield.upper_inf` allow
```
Comment (pope1ni) [inline docs/releases/3.1.txt:163]: `allow` → `lookups allow`

**Comment by felixxm** [comment]
@carltongibson Thanks :+1:

**Pull request merged**